moveit_ros/benchmarks/examples/demo_obstacles.yaml

```diff
@@ -0,0 +1,33 @@
+# This is an example configuration that loads the "Kitchen" scene from the
+# local MoveIt warehouse and benchmarks the "manipulator" group over all pairs
+# of motion plan queries and start states in the Kitchen scene.
+
+# Five planners from two different plugins are run a total of 50 times each, with a
+# maximum of 10 seconds per run.  Output is stored in the /home/benchmarks directory.
+
+benchmark_config:
+    warehouse:
+        host: 127.0.0.1
+        port: 33829
+        scene_name: ObstaclesScene     # Required
+    parameters:
+        name: ObstaclesScenePick1
+        runs: 50
+        group: panda_arm      # Required
+        timeout: 10.0
+        output_directory: /tmp/moveit_benchmarks/
+        queries: .*
+        start_states: .*
+    planners:
+        - plugin: ompl_interface/OMPLPlanner
+          planners:
+            - RRTConnectkConfigDefault
+            - BKPIECEkConfigDefault
+            - KPIECEkConfigDefault
+        - plugin: chomp_interface/CHOMPPlanner
+          planners:
+            - CHOMP
+        - plugin: stomp_moveit/StompPlannerManager
```

Comment:
maybe `stomp_moveit` should be renamed to `stomp_interface` to be consistent with OMPL and CHOMP?

Reply:
I'm guessing this is tied by the package name

Reply:
yes this is the package name: `stomp_moveit`

Reply:
My point was that perhaps the package should be renamed for consistency. This can be done later, if others agree.

Reply:
i agree it would be nice